rust/projects/file-io/project.md

```diff
@@ -300,6 +307,53 @@ additional information to distinguish the length of each record. Maybe not.
 
 _Implement "get" now_.
 
+Be sure you pass the tests before submitting Part 3, you need to pass all the tests about `get`:
+
+```
+cargo test --test tests -- get
+...
+
+running 5 tests
+test get_non_existent_value ... ok
+test get_stored_value ... ok
+test cli_get_non_existent_key ... ok
+test cli_invalid_get ... ok
+test cli_get_stored ... ok
```

Comment:
In general, can you rename the tests so that they are prefixed in a way that distinguishes them. So here there's `cli_invalid_get`. I'd rather that be `cli_get_invalid`. Having a systematic naming scheme will make things much more predictable for the user.

I'm ok with "namespacing" the lib vs cli tests as here, though I might suggest putting `lib_` in front of the library tests.

Reply:
cc @sticnarf what do you think about the idea of namespacing the tests as `lib_` and `cli_`?

Reply:
hm, once we get to the networking tests, they are namespaced by their test suits it seems - i.e. there's a `kv_store.rs` test file.

Reply:
I've proposed another organizational scheme elsewhere in my review comments.